sdk/src/test/java/com/hedera/hashgraph/sdk/TokenAssociateTransactionTest.java

```diff
@@ -84,4 +86,57 @@ void fromScheduledTransaction() {
 
         assertThat(tx).isInstanceOf(TokenAssociateTransaction.class);
     }
+
+    @Test
+    void ConstructTokenDeleteTransactionFromTransactionBodyProtobuf() {
+        var accountId = AccountId.fromString("1.2.3");
+        var tokenIds = List.of(TokenId.fromString("4.5.6"),
+            TokenId.fromString("7.8.9"),
+            TokenId.fromString("10.11.12"));
+
+        var transactionBody = TokenAssociateTransactionBody.newBuilder()
+            .addAllTokens(tokenIds.stream().map(TokenId::toProtobuf).toList())
+            .setAccount(accountId.toProtobuf()).build();
+        var txBody = TransactionBody.newBuilder().setTokenAssociate(transactionBody).build();
+        var tokenAssociateTransaction = new TokenAssociateTransaction(txBody);
+
+        assertThat(tokenAssociateTransaction.getAccountId()).isEqualTo(accountId);
+        assertThat(tokenAssociateTransaction.getTokenIds()).isEqualTo(tokenIds);
+    }
+
+    @Test
+    void getSetAccountId() {
+        var accountId = AccountId.fromString("1.2.3");
```

Comment:
Should we make some rule for unit tests that we instantiate such vars at the top of the file (like in C++ SDK or in my PRs)?
Here's couple reasons of that:
- code consistency (we're syncing w/ @dikel on couple guideline rules, so we make almost identical PRs for unit tests)
- less repeatable code

but .. we just can refactor it afterwards as well, so would be waiting here for POV of other reviewers:)

Reply:
I agree with that. We should follow the `DRY (Don't Repeat Yourself)` principle where it's possible.